gfx_imgui.h

```diff
@@ -319,12 +298,13 @@ class GfxImGuiInternal
 
         io.MouseWheel  = mouse_wheel;
         io.MouseWheelH = mouse_wheelh;
-        if(g_hWnd != 0)
+        ImGui_ImplWin32_Data* bd = ImGui_ImplWin32_GetBackendData();
+        if(bd != nullptr && bd->hWnd != 0)
```

Comment:
Theres no global ```g_hWnd``` anymore in the win32 backend as everything is now encapsulated inside ```ImGui_ImplWin32_Data```